utils/DiscordBot.ts

```diff
@@ -1,6 +1,8 @@
 import * as Discord from 'discord.js'
 
-export const DiscordBot = new Discord.Client()
+export const DiscordBot = new Discord.Client({
+	intents: 32767
```

Comment:
컨피그에서 커스터마이징 하는게 좋아보여요 approve하시면 `.env.demo.local` 에 추가해주세요!
```suggestion
	intents: Number(process.env.DISCORD_CLIENT_INTENTS) ?? 32767
```